src/azure-cli-core/azure/cli/core/tests/test_help.py

```diff
@@ -486,10 +506,18 @@ def test_load_from_help_json(self, mocked_load, mocked_pkg_util, mocked_getmembe
         self.assertEqual(obj_param_dict["--arg3"].value_sources[2]['link'],
                          {"command": "az test show", "title": "Show test details. Json file"})
 
-        self.assertEqual(command_help_obj.examples[0].short_summary, "A simple example from json")
-        self.assertEqual(command_help_obj.examples[0].long_summary, "More detail on the simple example.")
-        self.assertEqual(command_help_obj.examples[0].command, "az test alpha --arg1 alpha --arg2 beta --arg3 chi")
-        self.assertEqual(command_help_obj.examples[0].supported_profiles, "2018-03-01-hybrid, latest")
+        if self.test_cli.cloud.profile in ['2018-03-01-hybrid', 'latest']:
+            self.assertEqual(command_help_obj.examples[0].short_summary, "A simple example from json")
+            self.assertEqual(command_help_obj.examples[0].long_summary, "More detail on the simple example.")
+            self.assertEqual(command_help_obj.examples[0].command, "az test alpha --arg1 alpha --arg2 beta --arg3 chi")
+            self.assertEqual(command_help_obj.examples[0].supported_profiles, "2018-03-01-hybrid, latest")
+
+        if self.test_cli.cloud.profile == '2019-03-01-hybrid':
+            # only supported example here
+            self.assertEqual(len(command_help_obj.examples), 0)
```

Comment:
should it be 1?

Reply:
The last and only effective help entries is placed at `set_help_json`, its supported to show in profile `2018-03-01-hybrid` and `latest`:
https://github.com/Azure/azure-cli/blob/fdc155cf7aba336ea2bf930372053c1e0adce7d4/src/azure-cli-core/azure/cli/core/tests/test_help.py#L308-L314

That's why it's 1 under profile 2019